mitodl/open-discussions
# Updated README and added references to common web app guide

#### Pre-Flight checklist


#### What are the relevant tickets?
Closes #41 

#### What's this PR do?
Overhauls the README to correct various setup steps and adds references to the [common web app guide](https://github.com/mitodl/handbook/blob/master/common-web-app-guide.md)

#### How should this be manually tested?
If you _really_ want to test this, run through the full README from a completely clean slate and check that it works.

#### Any background context you want to provide?

I'm going to create a separate issue to move the giant code block for setting up a new channel into a management command.
